content/docs/command-reference/repro.md

```diff
@@ -244,8 +249,8 @@ If we now run `dvc repro`, we should see this:
 ```dvc
 $ dvc repro
 Stage 'filter' didn't change, skipping
-Running stage 'count' with command:
-        python process.py numbers.txt > count.txt
+Running stage 'count':
```

Comment:
O I removed "with command" from this file but there's another 4 instances/files with this text in the repo.